Inputs/Input_Info.py

```diff
@@ -7,7 +7,8 @@
 
 ################################
 # Name of output directories
-datacardInputs = "datacardInputs"
+datacardInputs = "{year}/datacardInputs"
```

Comment:
If we add {year} here, then we need to propagate this info to the efficiency module

```bash
Traceback (most recent call last):
  File "efficiencyFactors.py", line 947, in <module>
    if not os.path.isdir(opt.ERA+"/"+datacardInputs): os.mkdir(opt.ERA+"/"+datacardInputs)
OSError: [Errno 2] No such file or directory: '2018/{year}/datacardInputs'
```